lib/src/ast/element.dart

```diff
@@ -97,6 +109,20 @@ abstract class ElementAst implements StandaloneTemplateAst {
   /// Whether this is a `<template>` tag and should not be directly rendered.
   bool get isEmbeddedTemplate => name == 'template';
 
+  /// Determines whether the element tag name is void element.
+  bool get isVoidElement;
+
+  /// Whether this uses the void tag end '/>'.
+  ///
+  /// By definition, a void element can use either '>' or '/>'.
+  bool get usesVoidTagEnd;
```

Comment:
is it possible to get `/ >` and do we treat that the same? In other words, do we need an offset for this?